payments-app-extension-credit-card/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,28 @@
+# The version of APIs your extension will receive. Learn more:
+# https://shopify.dev/docs/api/usage/versioning
+api_version = "2021-07"
+
+[[extensions]]
+name = "{{ name }}"
+type = "payments_app_extension"
+handle = "{{ handle }}"
+
+[[extensions.targeting]]
+target = "credit_card"
```

Comment:
Not sure babout this syntax, looks like the target is missing the initial part (what is targeted)

<img width="657" alt="Screenshot 2024-01-10 at 14 40 32" src="https://github.com/Shopify/extensions-templates/assets/4012836/fbdb891c-4887-403a-b9a2-ba81c66ef827">

You can find some examples here: https://github.com/Shopify/extensions-templates/blob/main/checkout-extension/shopify.extension.toml.liquid

Reply:
Hey @raluka, thanks for sharing. I believe the concept of "target" for a payments extension differs from that of a UI or Function target, in that the "target" represents the specific type of payment extension. In moving the payment extension to the CLI we are replacing the 5 dashboard configurations in core with a single CLI configuration in core. Therefore for payment extensions I see something like:
```
a. What's targeted          |           b. Action
payment type                |           shared payment verbs + specific verbs depending on payment type

e.g.
1. offsite
2. custom onsite
3. creditcard
4. custom creditcard
5. redeemable
```

Reply:
Also, the problem with using the 
`[[extensions.targeting]]` 
convention is that it requires a [module parameter](https://shopify.dev/docs/apps/app-extensions/configuration#targets) which does not apply to us